README.md

```diff
@@ -568,59 +616,58 @@ document.addEventListener("pjax:success", whenDOMReady)
 
 _Note: Don't create the Pjax instance in the `whenDOMReady` function._
 
-For my concern and usage, I `js-Pjax`-ify all body children, including stuff like navigation and footer (to get navigation state easily updated).
+For my concern and usage, I `js-Pjax`-ify all body children, including stuff like navigation and footer (to update navigation state).
```

Comment:
I'm not sure that this or the line below is useful, so I would be inclined to remove them.